avocado/etc/avocado/avocado.conf

```diff
@@ -14,7 +14,7 @@ logs_dir = ~/avocado/job-results
 
 [sysinfo.collect]
 # Whether to collect system information during avocado jobs
-enabled = True
+enabled = on
```

Comment:
@narasimhan-v, 

Actually, AFAIK, this is not necessary as `register_option()` is saving the default values now.

There is a new PR (#3934) to remove the templates files. Can you please check if this will work when there is no avocado.conf files?

Reply:
@beraldoleal Yes, this will work when there is no conf files

Reply:
Ok, so I think that we can close this and wait for #3934 to be approved. What do you think?

Reply:
Sure, closing.